package.json

```diff
@@ -36,6 +36,7 @@
     "lint:scss": "yarn lint:scss:precommit '**/*.scss'",
     "lint:scss:precommit": "stylelint --fix --ignore-path .gitignore",
     "lint-staged": "lint-staged",
+    "postinstall": "npx @carbon/telemetry@0.0.0-alpha.6 collect --install",
```

Comment:
@joshblack What are your thoughts on this approach? We're hoping to reduce the amount of additional dependencies that land in applications

Reply:
@SimonFinney definitely go for it! I think if the `npx` approach works then we would want to use it too, honestly.